packages/paste-website/src/pages/index.tsx

```diff
@@ -46,6 +47,27 @@ const IndexPage: React.FC<{}> = (): React.ReactElement => {
             View Components
           </Button>
         </Box>
+        <Callout>
+          <CalloutTitle as="h4">Looking for Flex / Console / SendGrid design systems?</CalloutTitle>
+          <CalloutText>
+            Paste is in early access. For now, you will still need to use your current design systems as well. More
+            information about this will come in a future release.
+          </CalloutText>
+          <CalloutText marginTop="space40">
```

Comment:
Can you add `as="div"` to this `CalloutText`. Right now it's producing a list inside of a paragraph and that's not valid HTML.